.github/actions/install-python-env/action.yml

```diff
@@ -11,6 +11,8 @@ runs:
   using: "composite"
   steps:
     - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
Consider exploring making this action more flexible about fetch-depth through parameterization (I feel this may be work outside the scope of this PR).

Reply:
I agree! But I'm not sure what you mean by using parameterization. Would this be added through the inputs object and then called through `entropy-check.yml`? What is the benefit for this action to not grab all the commits?